### DISCUSSION TITLE
Searchable options in the SelectColumn

### DISCUSSION THREAD

**Discussion by borna453** [filamentphp/filament#14589] [Ideas]
@danharrin Hi, I was recently working with filament, and specifically the SelectColumn. I noticed that it doesnt have the same searchable option like the Select form field. Is there something filament related thats preventing this being a feature or was there no need for it? 
I did end up making a custom SearchableSelectColumn, where i extended the default SelectColumn, and edited the view.
It works great and i was wondering if this would be something possible to implement? For me it would be useful across multiple projects.

Example of how it looks(it works exaclty like the SelectColumn):
![image](https://github.com/user-attachments/assets/fb28310e-f36e-4f58-9b7b-0c89e69468c8)

**Comment by danharrin** [comment]
Hi, I know this is a common community request but it's not something that is currently on the roadmap. But I will consider it again in the future. If I do it, I'd want it to match the features of the Select, and currently that would be very tricky as columns don't have the same underlying features as form fields to power that sort of thing.
